src/controller/python/CHIPDeviceController-PyBind.h

```diff
@@ -0,0 +1,55 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+#pragma once
+#include <controller/CHIPDevice.h>
+#include <controller/CHIPDeviceController.h>
+#include <core/CHIPCore.h>
+#include <setup_payload/QRCodeSetupPayloadParser.h>
+#include "PyDelegateWrappers.h"
+
+namespace chip {
+namespace Controller {
+
+class CHIPDeviceControllerPyBind
```

Comment:
We need to move interfaces to be pythoning and user-friendly.

This class is functional fast, however it makes concessions for this: it  is a flat namespace for several items (commisioner, nodeids, delegates,  params) and feels like raw C++ (returns CHIP_ERROR, functions seem very raw)

I would expect more control on things like 'logging' controls logging (e.g. can setup python redirects), things like 'commisioning' handles commisioning, working with devices has classes of the form 'device'.